### PR TITLE
Actualization project template

### PR DESCRIPTION
**Pull request by MikhailKalyn**

### CODE REVIEW

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/StepDefinitions/ContactUsPageSteps.cs [hunk 1]
```diff
@@ -48,5 +54,12 @@ public void ThenNotificationAboutEmptyFieldsIsPresent()
             Assert.IsTrue(contactUsPage.IsEmailValidationMessagePresent,
                 "Email validation message should be displayed");
         }
+
+        [Then(@"Contact us page dump is difference")]
```
Comment (n-verbitsky) [inline Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/StepDefinitions/ContactUsPageSteps.cs]: Shouldn't it be "Contact us page dump is different"?

**Review by n-verbitsky — APPROVED** [review]

File: Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/StepDefinitions/ContactUsPageSteps.cs (hunk 1 above)
Comment (MikhailKalyn) [reply Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/StepDefinitions/ContactUsPageSteps.cs]: done

**Comment by mialeska** [comment]
Please fix tests, they should not fail in pipeline

File: Aquality.Selenium.Template/Aquality.Selenium.Template/Resources/settings.json [hunk 2]
```diff
@@ -1,94 +1,148 @@
 {
-  "environment": "Stage",
-  "browserName": "chrome",
-  "isRemote": false,
-  "remoteConnectionUrl": "http://localhost:4444/wd/hub",
-  "isElementHighlightEnabled": true,
+    "environment": "Stage",
+    "browserName": "chrome",
```
Comment (mialeska) [inline Aquality.Selenium.Template/Aquality.Selenium.Template/Resources/settings.json:3]: please ensure that tests are passed in the pipeline

**Review by mialeska — CHANGES_REQUESTED** [review]

File: Aquality.Selenium.Template/Aquality.Selenium.Template/Resources/settings.json (hunk 2 above)
Comment (mialeska) [reply Aquality.Selenium.Template/Aquality.Selenium.Template/Resources/settings.json:3]: fixed

**Pull request merged**